_centers/dyer-county-humane-society.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: center
+title: Dyer County Humane Society
+phone: 731-285-4887
+address: 1120 East Court
+city: Dyersburg
+state: TN
+zip: 38024
+primary: sheets towels stuffed toys
```

Comment:
Can we narrow down the primary category to a single generic category, like `fabric` perhaps?